typescript-operators-and-expressions/main.ts

```diff
@@ -0,0 +1,45 @@
+const width: number = 10;
+const height: number = 10;
+const area: number = width * height;
+
+console.log('value of area:', area);
+console.log('type of area:', typeof area);
+
+const bill: number = 120;
+const payment: number = 140;
+const change: number = bill - payment;
```

Comment:
`payment` & `bill` should be swapped. Hopefully you pay enough to cover your bill 😉

Reply:
hope this worked